Articles/Blog/2020-03-maintainers-drive-sustainability.md

```diff
@@ -8,60 +8,60 @@
+The impact of a maintainer can be seen in the history of commits to the ParFlow repository in Figure 3. During the period from spring  2011 through fall  2015 ParFlow lost support for a maintainer, and the number of commits being made to the main code base dropped dramatically.   When maintainer support was reintroduced, the repository activity picked up significantly.   Development had not stopped, but little effort had been made to integrate changes into the main repository, and the enhancements were not being made available for the broader community.
 
 [Figure 3: History showing number of code commits to ParFlow repository during periods with and without a maintainer]<img src='https://github.com/betterscientificsoftware/images/raw/master/maintainers_image_0.png' class='page lightbox' />
 
-> As hydrologist without formal training in computer science, or dedicated resources for code maintenance, the role of the maintainer is a critical factor in my ability to both use and contribute to ParFlow.  The code maintainer has made it possible for me to follow best practices in my research and to integrate the changes I make into the main branch and share with the community.  **Laura E. Condon, Assistant Professor, Hydrology and Atmospheric Sciences, University of Arizona**
+> As a hydrologist without formal training in computer science or dedicated resources for code maintenance, the role of the maintainer is a critical factor in my ability to both use and contribute to ParFlow.  The code maintainer has made it possible for me to follow best practices in my research and to integrate the changes I make into the main branch and share with the community.  **Laura E. Condon, Assistant Professor, Hydrology and Atmospheric Sciences, University of Arizona**
       
-All of these practices and tools have made the role of maintainer easier, more distributed, more automated, and more sustainable with limited budgets.  Will ParFlow be around in another 25 years?  We have no idea but then no-one is more surprised at ParFlow’s longevity than the original developers.   You never know where a project might take you when you start.
+All of these practices and tools have made the role of maintainer easier, more distributed, more automated, and more sustainable with limited budgets.  Will ParFlow be around in another 25 years?  We have no idea, but then no-one is more surprised at ParFlow’s longevity than the original developers.   You never know where a project might take you when you start.
 
 **More about ParFlow:**
 
```

Comment:
Do we have a link or other info for this?

Reply:
@prwolfe there are links to the parflow web site and GH repo in the table below.  We can make them clickable.  Does that address your question?  Or are you looking for something else?

Reply:
Looks much better - Thanks!